### PR TITLE
Pass PR number to privileged job.

### PR DESCRIPTION
**Pull request by bwplotka**

### CODE REVIEW

**Comment by bwplotka** [comment]
Tmp fork until https://github.com/pozil/auto-assign-issue/pull/115 is merged.

File: .github/workflows/auto-assign-issue.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: (Privileged) Auto Assign
```
Comment (TheSpiritXIII) [inline .github/workflows/auto-assign-issue.yml:1]:
```suggestion
name: Auto Assign Issue
```
This one isn't privileged.

Comment (bwplotka) [reply .github/workflows/auto-assign-issue.yml:1]: It still is (:

Comment (bwplotka) [reply .github/workflows/auto-assign-issue.yml]: but yea, no biggie

File: .github/workflows/auto-assign-pr.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success'
+    steps:
+    - name: 'Download parameter'
+      uses: actions/github-script@v3.1.0
+      with:
+        script: |
+          var artifacts = await github.actions.listWorkflowRunArtifacts({
```
Comment (TheSpiritXIII) [inline .github/workflows/auto-assign-pr.yml:22]: I would use `const` here in place of all the `var`s.

`var` isn't really used in JavaScript anymore (`let` is for mutable references).

**Review by TheSpiritXIII — APPROVED** [review]
Thanks!

File: .github/workflows/auto-assign-pr.yml (hunk 2 above)
Comment (bwplotka) [reply .github/workflows/auto-assign-pr.yml:22]: Let me fix this later, want to first try if this is a right path

**Pull request merged**